roles/openshift_router/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# TODO: Centralized openshift_config openshift_facts??
+# TODO: anchor this on a master
```

Comment:
What do you mean by anchor on master?  If you mean install on master, then it would require us to update bin/cluster to provision the master as a node.

What we really want to do is to label a node as an "infra" node and land the router and registry there: https://github.com/openshift/training/blob/master/beta-3-setup.md#regions-and-zones